insights/client/client.py

```diff
@@ -171,63 +105,107 @@ def register(config):
     if not username and not password and not auto_config and authmethod == 'BASIC':
         logger.debug('Username and password must be defined in configuration file with BASIC authentication method.')
         return False
-    pconn = get_connection(config)
     return pconn.register()
 
 
-def handle_registration(config):
-    """
-        returns (json): {'success': bool,
-                        'machine-id': uuid from API,
-                        'response': response from API,
-                        'code': http code}
-    """
+def handle_registration(config, pconn):
+    '''
+        Handle the registration process
+        Returns:
+            True - machine is registered
+            False - machine is unregistered
+            None - could not reach the API
+    '''
     # force-reregister -- remove machine-id files and registration files
     # before trying to register again
-    new = False
     if config.reregister:
-        logger.debug('Re-register set, forcing registration.')
```

Comment:
should restore the missing log messages in this function

Reply:
do you need to follow up with this?

Reply:
Already did. Added in a commit either this morning or yesterday.